css-three-column-layout/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Three Column Layout</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h1>
+          Designed for life science. Optimized for SaMD and Digital Health.
+        </h1>
+      </div>
+
+      <div class="row">
+        <div class="column-one-third column-container">
+          <div class="img-container">
+            <div>
+              <img src="images/merge.png" alt="merge" />
```

Comment:
In the future, you _could_ make this more specific and reusable.

You could split this into a `column` class which is `display: flex; flex-direction: column` and an `items-center` class that is `align-items: center;`

This way in your project if you needed more columns or even if you need to center align a row, you can reuse it.